### PR TITLE
Update markdown dependency to allow reverting the uri rewrite.

### PR DESCRIPTION
**Pull request by domesticmouse**
Also explicitly mark this package as requiring Flutter 1.10.7.

### CODE REVIEW

**Comment by KagurazakaHanabi** [comment]
v1.10.7 also is fine（？）

File: pubspec.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ dev_dependencies:
 
 environment:
   sdk: '>=2.5.0 <3.0.0'
+  flutter: ^1.11.0
```
Comment (KagurazakaHanabi) [inline pubspec.yaml]: This line should like `flutter: ">=1.10.7 <2.0.0"`？

Comment (domesticmouse) [reply pubspec.yaml]: I can do that.

**Comment by domesticmouse** [comment]
PTAL @KagurazakaHanabi =)

**Comment by domesticmouse** [comment]
Explicit version in this release for https://github.com/flutter/flutter_markdown/issues/156 and https://github.com/flutter/flutter_markdown/issues/159

**Pull request merged**